RFS/src/test/java/com/rfs/framework/ElasticsearchContainer.java

```diff
@@ -0,0 +1,61 @@
+package com.rfs.framework;
+
+import java.time.Duration;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Containerized version of Elasticsearch cluster
+ */
+public class ElasticsearchContainer implements AutoCloseable {
```

Comment:
This is cool

Reply:
Is there a reason not to use the testcontainer version of ES as a starting point